domains/kushal.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kushalshah0",
+        "email": "code@kushalshah.com.np"
+    },
+    "record": {
+        "CNAME": "portfolio.kushalshah.com.np"
```

Comment:
Unable to access site.

Reply:
(https://portfolio.kushalshah.com.np)
this site is accessible sir.